KnownMimeTypes.tt

```diff
@@ -16,13 +16,11 @@ using System;
 namespace MimeMapping
 {
 <#
-    const string APACHE_URL = "http://svn.apache.org/repos/asf/httpd/httpd/trunk/docs/conf/mime.types";
-    const string NGINX_URL = "https://raw.githubusercontent.com/h5bp/server-configs-nginx/master/mime.types";
-	const string MIMEDB_URL = "https://raw.githubusercontent.com/jshttp/mime-db/master/src/custom-types.json";
+	const string MIMEDB_URL = "https://cdn.jsdelivr.net/gh/jshttp/mime-db@master/db.json";
```

Comment:
Looking at the readme for that repo:
> If you're crazy enough to use this in the browser, you can just grab the JSON file using [jsDelivr](https://www.jsdelivr.com/). 

In our case, we're grabbing this data in a source generation script which is rarely ran. In that case, probably makes more sense to grab directly from github as opposed to this cdn.

> It is recommended to replace master with [a release tag](https://github.com/jshttp/mime-db/tags) as the JSON format may change in the future.

It sounds like we should also follow this advice, so this repo doesn't randomly break in the future if the format changes. And a specific tag will make the set of mime types supported by this repo will be more clear.